scratch/setup.py

```diff
@@ -1,4 +1,4 @@
-from distutils.core import setup, Extension
```

Comment:
Are you using this? The scratch space is kind of a dump for ideas that I never care to clean up. :smile:

Reply:
I only use the toplevel setup.py for deploying packages.

Reply:
Nope, just clearing out imports. Probably worth deleting and hiding in `.gitignore`